singlem/pipe.py

```diff
@@ -304,9 +307,13 @@ def return_cleanly():
                 # outputs
                 otu_table_object,
                 package_to_taxonomy_bihash)
-
         if len(transcript_tempfile_name_to_desired_name) > 0:
             otu_table_object.rename_samples(transcript_tempfile_name_to_desired_name)
+        if not include_off_target_hits:
+            collection = OtuTableCollection()
+            collection.otu_table_objects.append(otu_table_object)
+            packages = [SingleMPackage.acquire(path) for path in singlem_packages]
```

Comment:
This acquisition already happens above, confusingly the variable is names `hmms`. I think you can just iterate that instead of acquiring again.